README.md

```diff
@@ -1,2 +1,36 @@
-# pgbouncer-container
-A docker container, containing the pgbouncer connection pooler. Built for use in the pgbouncer k8s charm: https://github.com/canonical/pgbouncer-k8s-operator
+# PgBouncer Container
+
+This docker image defines and configures the [PgBouncer connection pooler for PostgreSQL](https://www.pgbouncer.org), with a set of sensible defaults in pgbouncer.ini. These defaults are insecure, and should be modified to suit your needs, specifically in pgbouncer.ini and userlist.txt.
+
+Built for use in the pgbouncer k8s charm: https://github.com/canonical/pgbouncer-k8s-operator
+
+## Usage, and connecting to a postgres container
+
+1 in one terminal, run the following:
+  1 `docker pull postgres/postgres`
+  1 `docker run --rm -P -e POSTGRES_PASSWORD=pw --name pg --hostname pg-host postgres`
+1 in another terminal:
+  1 `docker build -t pgbouncer:local .`
+  1 `docker run -rm --name pgbouncer:local`
```

Comment:
It should be changed to `docker run --rm --name pgbouncer pgbouncer:local` (missing a dash in the rm flag and missing container name).

Reply:
I've updated the code to match this. I'm also adding an `--env-file` option, since it's the easiest way to pass in these options, and `-it` tag, since pgbouncer doesn't have an easy way to exit from the container in this way.